examples/specs/bar_layered_opacity.json

```diff
@@ -0,0 +1,25 @@
+{
+  "description": "A bar chart showing the US population distribution of age groups and gender in 2000.",
+  "data": { "url": "data/population.json"},
+  "transform": {
+    "filter": "datum.year == 2000",
+    "calculate": [{"field": "gender", "expr": "datum.sex == 2 ? \"Female\" : \"Male\""}]
+  },
+  "mark": "bar",
+  "encoding": {
+    "x": {
+      "field": "age", "type": "ordinal",
+      "scale": {"bandSize": 17}
+    },
+    "y": {
+      "aggregate": "sum", "field": "people", "type": "quantitative",
+      "axis": {"title": "population"}
+    },
+    "opacity": {
+      "field": "gender", "type": "nominal"
+    }
+  },
+  "config": {
+    "mark": {"opacity": 0.6, "stacked" : "none"}
```

Comment:
I think I will just change this example to demonstrate something else since the old one doesn't make sense.